packages/core/data-transfer/src/engine/index.ts

```diff
@@ -903,6 +911,9 @@ class TransferEngine<
 
   async transferConfiguration(): Promise<void> {
     const stage: TransferStage = 'configuration';
+    if (this.shouldSkipStage(stage)) {
```

Comment:
I think I did this to prevent it from being called even directly. I think it works for now. In the future, I can potentially see a use case for the transfer engine skipping it, but allowing it to be called manually, and we can update it then to allow in that case.